crowbar_framework/config/routes.rb

```diff
@@ -75,6 +75,12 @@
   post "utils/repositories/deactivate(.:format)", controller: "repositories", action: "deactivate", as: "deactivate_repository"
   post "utils/repositories/activate_all(.:format)", controller: "repositories", action: "activate_all", as: "activate_all_repositories"
   post "utils/repositories/deactivate_all(.:format)", controller: "repositories", action: "deactivate_all", as: "deactivate_all_repositories"
+  get "utils/backup", controller: "backup", action: "index", as: "backup"
```

Comment:
I would prefer a `resources` definition instead of all those separate definitions.